samples/ComputeSharp.SwapChain/Shaders/FourColorGradient.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace ComputeSharp.SwapChain.Shaders;
```

Comment:
Moving this to file-scoped namespaces in a separate commit creates issues with git blame. Please either add the commit where you did so in the `.git_ignore_revs` file, or (better) just squash this commit on the one where you added this file in the first place directly.

Reply:
Fixed